README.md

```diff
@@ -177,7 +177,7 @@ To run, build and test locally:
     * If npm is not installed install [nvm](https://github.com/nvm-sh/nvm#intro) in root directory
```

Comment:
Could you just put a note about running the jest tests in the README. There is a section about running the tests for sbt, so maybe rework that to cover both sorts of tests.

Reply:
I've added in a line about running `npm test` from the root directory.